@ecomplus/storefront-components/src/html/CartQuickview.html

```diff
@@ -102,6 +102,21 @@
           >
             {{ i19seeCart }}
           </a-link>
+
+          <button
+            type="button"
+            class="minicart__btn-cart btn btn-block btn-outline-secondary close d-md-none"
```

Comment:
```suggestion
            class="minicart__btn-back btn btn-sm btn-link d-md-none"
```

`minicart__btn-cart` é pro botão "ir para o carrinho", `close` é uma classe do Bootstrap específica para botões com o "x" pra fechar modai, offsets, etc...

Reply:
Pra ficar diferente do outro botão (e menos destacado) sugeri `btn-link` também